camel-knative/camel-knative-http/src/main/java/org/apache/camel/component/knative/http/KnativeHttpConsumerDispatcher.java

```diff
@@ -167,6 +168,15 @@ protected void doStop() throws Exception {
 
         @Override
         public void handle(HttpServerRequest request) {
+            if (request.method() != HttpMethod.POST) {
+                HttpServerResponse response = request.response();
+                response.setStatusCode(405);
+                response.putHeader(Exchange.CONTENT_TYPE, "text/plain");
+                response.end("Unsupported method: " + request.method());
```

Comment:
Not sure if we need to ask people to manually set "POST" or we should do it for them.
They're sending data to a `knative` endpoint that wraps HTTP after all.

I'm thinking to simple bridging integrations:
```
rest().put("/person/{id}").to("knative:event/person")
```

Reply:
the component does it by default, in your case you want to "put" an event to the broker and the underlying transport implementation will hit a POST to the target knative service.